run.py

```diff
@@ -33,6 +33,20 @@
     help="Do not drop last validate batch if "
     "if it is smaller than batch_size.",
 )
+@click.option(
+    "--affinity",
+    "-af",
+    type=str,
+    default=None,
+    help="Path to affinity matrix for training.",
+)
+@click.option(
+    "--classes",
+    "-cl",
+    type=str,
+    default=None,
+    help="Path to a CSV file containing a list of classes for training.",
```

Comment:
Same as the comment above, need to be consistent if is a path or filename.

Reply:
In this case it actually refers to the full path on [line 159](https://github.com/alan-turing-institute/affinity-vae/blob/705f4850990165f5bb7faa6bc4db75be1d4db89d/avae/data.py#L159). I'll leave it at that since it is now consistent with affinities with the upcoming patch.